### ISSUE TITLE
Include a list of posts by the author on the bio page somewhere

### ISSUE THREAD

**Issue by kbaley** [westerndevs/western-devs-website#33]

**Comment by dylan-smith** [comment]
I really want to see list of posts on the bio pages!

**Comment by dpaquette** [comment]
This came up again today. Would be great to have a list of posts on the author page.

**Comment by kbaley** [comment]
Yeah this would be great! But it looks like we'll need someone who knows how to program computers and that has push access to the repository to get this done. I can't think of anyone here that fits that criteria.

**Comment by dlussier** [comment]
I'll set up an Access database for it...

**Comment by dpaquette** [comment]
I was thinking SharePoint would be a better solution

**Comment by davidwesst** [comment]
I've done this recently for my custom blog theme. Without looking too deep into it, using the [post object](https://hexo.io/docs/variables.html) and filtering by our custom property (authorId) would likely do the trick.

I'll try and take a look sooner rather than later, but there's this conference coming up and I want to make sure I do well there first. :)
